tutorials/gpu-deployment/gpu-deployment-from-scratch.md

```diff
@@ -0,0 +1,517 @@
+# GPU Deployment Guide 
+Guide Adapted from:[**EuroSciPy 2025**](https://euroscipy.org/schedule/) | [Share Link](https://jacobtomlinson.dev/links/euroscipy-2025)
+
+## Agenda
+- Deployment (40 mins)
```

Comment:
We should probably change this to be more representative.

Reply:
```suggestion
- Deployment
```